src/_pytest/logging.py

```diff
@@ -595,6 +602,15 @@ def __init__(self, config: Config) -> None:
         )
         self.log_cli_handler.setFormatter(log_cli_formatter)
 
+        if config.option.suppress_logger:
+            self._suppress_loggers()
```

Comment:
I would pass here the `config.option.suppress_logger` as ``_suppress_logger()` argument,
to isolate the later from object's state (ie avoid accessing `self._config.option.suppress_logger` twice from 2 different methods), and make this method reusable also from other sources (ie `conftest.py`).

Reply:
Yeah, that totally makes sense.